### PR TITLE
get-entries-filter.md

### PR DESCRIPTION
**Pull request by tschanfeld**
Extended description of the getEntries filter mentioned in Script Helper

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31582
fixes: https://github.com/demisto/etc/issues/37903

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by fvigo** [comment]
@tschanfeld this is very interesting, I believe it should be under the Development documentation section, not the Articles section.

CC @ShahafBenYakir  WDYT?

**Comment by tschanfeld** [comment]
I submitted it through this page - https://github.com/demisto/content-docs/tree/master/content-repo/extra-docs/articles, but if there's another page I should use in the future for development documentation, let me know. I'm still new to adding topics to xsoar.pan.dev.

**Comment by ShahafBenYakir** [comment]
@fvigo make sense

**Review by LPettay — APPROVED** [review]

**Comment by Itay4** [comment]
@fvigo i dont think it really makes sense to put it under dev, its a builtin command
i think it fits better under the ref docs, perhaps we can add a section for builtin commands like we have for scripts

**Comment by fvigo** [comment]
@Itay4 @ShahafBenYakir your call. I don't think it should be an article either (we also have the "Demisto class" in articles and it doesn't belong). I'm in favor of either creating a new section for builtins and one for class reference (like Demsito class and CommonServerPython), or one that includes both.

**Comment by ShahafBenYakir** [comment]
Yeah we all agree it should not be in articles and dev is also not a good place. @fvigo Do you recall if we currently have other pages that would fit `Builtins`?

**Comment by fvigo** [comment]
Not that I recall, but we should have them. Where does the "Script Helper" stuff that you can bring from the product UI live right now?

**Comment by ShahafBenYakir** [comment]
@fvigo It is hardcoded in the server code

**Comment by tschanfeld** [comment]
ideally, wherever this page goes, there will be a link to it from the Script Helper

**Comment by ShahafBenYakir** [comment]
@reutshal Let's talk and think about adding this new "Builtins" section

**Comment by tschanfeld** [comment]
Any decision where to put this?

**Comment by tschanfeld** [comment]
This pull request has been automatically marked as stale  ....

**Comment by ShahafBenYakir** [comment]
Closing until we will get the new section up

**Comment by ShahafBenYakir** [comment]
Waits 
https://github.com/demisto/etc/issues/37903

File: docs/integrations/get-entries-filter.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+id: get-entries-filter
+title: Get Entries Filter
+---
+When building a script, you can fetch entries from an incident. If you do not specify the incident id number, the script fetches from the current incident.  You have the option to create a filter to limit the search results.
```
Comment (yaakovi) [inline docs/integrations/get-entries-filter.md:5]:
```suggestion
When building a script, you can fetch entries from an incident. If you do not specify the incident id number, the script fetches from the current incident. You have the option to create a filter to limit the search results. 
```

**Review by yaakovi — APPROVED** [review]
💯

**Pull request merged**